package.json

```diff
@@ -59,6 +65,7 @@
     "through2": "^0.6.3"
   },
   "scripts": {
+    "start": "gulp server -o http://localhost:8080/dist/docs/index.html",
```

Comment:
The name of the command was `start` because that is a [standard command](https://docs.npmjs.com/cli/start) to start projects in Node, i.e. `npm start`. I was hoping that this would make the initial process of getting started with the project a little easier for some people. Either way, someone familiar with Node will still find this watch script and be able to run it via `npm runScript watch`.